manta/manta.go

```diff
@@ -191,8 +191,8 @@ func (c *Client) GetObject(path, objectName string) ([]byte, error) {
 	if err != nil {
 		return nil, errors.Newf(err, "failed to get object %s/%s", path, objectName)
 	}
-	res, _ := respData.RespValue.([]byte)
-	return res, nil
+	res, _ := respData.RespValue.(*[]byte)
```

Comment:
Given that this bug was originally hidden by discarding the "failed assertion boolean" in the first place maybe we should be checking it now to ensure there's no future regression?